classes/class-wc-connect-help-view.php

```diff
@@ -310,6 +322,50 @@ public function page() {
 			);
 		}
 
+		/**
+		 * @return array
+		 */
+		protected function get_tax_health_item() {
+			if ( class_exists( 'WC_Taxjar' ) ) {
+				return [
+					'state'              => 'warning',
+					'show_settings_link' => false,
+					'message'            => __( 'TaxJar extension detected. Automatic taxes functionality is disabled', 'woocommerce-services' ),
+				];
+			}
+
+			$store_country = WC()->countries->get_base_country();
+			if ( ! $this->taxjar_integration->is_supported_country( $store_country ) ) {
```

Comment:
Let's move this condition above the TaxJar one - if the country isn't supported anyway, let's not get hopes up by implying deactivating the TaxJar extension will help. :)